### ISSUE TITLE
Allow passing a list of packages with `"catalog:"` versions to the `consistentVersions` rule.

### ISSUE THREAD

**Issue by zerovox** [monorepolint/monorepolint#435]
Currently, calling:

```
import { consistentVersions } from "monorepolint/rules";
export default {
  rules: [
    consistentVersions({
      options: {
        matchDependencyVersions: {
             "@osdk/api": "catalog:",
        },
      },
    }),
  ],
};
```

Fails with the following:
```
monorepolint (mrl) had an unexpected error:
Error: Malformed expected package dependency version defined in monorepolint configuration: @osdk/api @ 'catalog:'
```
